src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -36,6 +42,37 @@
      * София - 900
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        Hashtable<String, Integer> employees = new Hashtable<>();
+        for (String value : names) {
+            employees.put(value, 0);
+        }
+
+        try {
+            DateFormat formatOfDate = new SimpleDateFormat("dd.mm.yyyy");
```

Comment:
we can make it as a constant

Reply:
what do you mean? make it final ? smth else?

Reply:
yes we can move it in the class:
```suggestion
            private static final DateFormat DATE_FORMAT = new SimpleDateFormat("dd.mm.yyyy");
```